src/platform_impl/linux/drm/window.rs

```diff
@@ -0,0 +1,354 @@
+            property::Value::SignedRange(0),
+        );
+        atomic_req.add_property(
+            event_loop_window_target.plane,
+            find_prop_id(
+                &event_loop_window_target.device,
+                event_loop_window_target.plane,
+                "CRTC_Y",
+            )
+            .expect("Could not get CRTC_Y"),
+            property::Value::SignedRange(0),
+        );
+        atomic_req.add_property(
+            event_loop_window_target.plane,
+            find_prop_id(
+                &event_loop_window_target.device,
+                event_loop_window_target.plane,
+                "CRTC_W",
+            )
+            .expect("Could not get CRTC_W"),
+            property::Value::UnsignedRange(event_loop_window_target.mode.size().0 as u64),
+        );
+        atomic_req.add_property(
+            event_loop_window_target.plane,
+            find_prop_id(
+                &event_loop_window_target.device,
+                event_loop_window_target.plane,
+                "CRTC_H",
+            )
+            .expect("Could not get CRTC_H"),
+            property::Value::UnsignedRange(event_loop_window_target.mode.size().1 as u64),
+        );
+
+        event_loop_window_target
+            .device
+            .atomic_commit(AtomicCommitFlags::ALLOW_MODESET, atomic_req)
+            .map_err(|_| {
+                crate::error::OsError::new(
+                    line!(),
+                    file!(),
+                    crate::platform_impl::OsError::DrmMisc("Failed to set mode"),
+                )
+            })?;
+
+        Ok(Self(
+            event_loop_window_target.mode.clone(),
+            event_loop_window_target.connector.clone(),
+            event_loop_window_target.event_loop_awakener.clone(),
+            event_loop_window_target.device.clone(),
+        ))
+    }
+    #[inline]
+    pub fn id(&self) -> super::WindowId {
+        super::WindowId
+    }
+
+    #[inline]
+    pub fn set_title(&self, _title: &str) {}
+
+    #[inline]
+    pub fn set_visible(&self, _visible: bool) {}
+
+    #[inline]
+    pub fn is_visible(&self) -> Option<bool> {
+        Some(true)
+    }
+
+    #[inline]
+    pub fn outer_position(&self) -> Result<PhysicalPosition<i32>, NotSupportedError> {
+        Err(NotSupportedError::new())
+    }
+
+    #[inline]
+    pub fn inner_position(&self) -> Result<PhysicalPosition<i32>, NotSupportedError> {
+        Err(NotSupportedError::new())
+    }
+
+    #[inline]
+    pub fn set_outer_position(&self, _position: Position) {}
+
+    #[inline]
+    pub fn inner_size(&self) -> PhysicalSize<u32> {
+        let size = self.0.size();
+        PhysicalSize::new(size.0 as u32, size.1 as u32)
+    }
+
+    #[inline]
+    pub fn outer_size(&self) -> PhysicalSize<u32> {
+        self.inner_size()
+    }
+
+    #[inline]
+    pub fn set_inner_size(&self, _size: Size) {
+        // It's technically possible to do this by changing video modes but that seems a little
```

Comment:
This shouldn't be done by changing video mode. Just state that it's impossible to do so.

Reply:
Fixed